### PR TITLE
fix #14: support Python3

### LINKED ISSUE #14: Python3 support

### PR DESCRIPTION
**Pull request by danni-m**

### CODE REVIEW

File: RAMP/commands_discovery.py [hunk 1]
```diff
@@ -81,6 +81,7 @@ def _get_redis_command_info(redis_client, command_name):
         Returns ModuleCommand
     """
     command_info = redis_client.execute_command("COMMAND INFO {}".format(command_name))
+    print("COMMAND INFO {}".format(command_name))
```
Comment (swilly22) [inline RAMP/commands_discovery.py]: Remove print.

File: unpack_test.py [hunk 2]
```diff
@@ -28,10 +28,8 @@ def test_valid_bundle():
     """
     test bundle extraction
     """
-    module_path = os.path.join(os.getcwd(), MODULE_FILE)
+    module_path = os.path.join(os.getcwd(), MODULE_FILE_PATH)
```
Comment (swilly22) [inline unpack_test.py]: MODULE_FILE_PATH already contains os.getcwd().

Comment (danni-m) [reply unpack_test.py]: Not exactly, the CWD is the root of the repository and the module is in `test_module` folder.

File: RAMP/commands_discovery.py (hunk 1 above)
Comment (danni-m) [reply RAMP/commands_discovery.py]: 👍

**Pull request merged**